src/app/shared/devices/device-list/device-list.component.ts

```diff
@@ -17,8 +18,9 @@ const headersForDisplay = [
 })
 export class DeviceListComponent {
     @Input() deviceList: Array<Device>;
+    @Input() editOnItemClick: boolean = true;
     @Input() showDeleteButton: boolean = true;
-    @Input() showStatisticLink: boolean = true;
+    @Input() statisticLink: string|boolean = false;
```

Comment:
!fix
Will be better to use only one type of variable and in *ngIf just check if length of string more than 0.